engine/runtime-instrument-repl-debugger/src/main/java/org/enso/interpreter/instrument/repl/debugger/ReplDebuggerInstrument.java

```diff
@@ -251,7 +251,7 @@ protected void onEnter(VirtualFrame frame) {
         CallerInfo lastScope = Function.ArgumentsHelper.getCallerInfo(frame.getArguments());
         Object lastReturn = EnsoContext.get(this).getNothing();
         // Note [Safe Access to State in the Debugger Instrument]
-        monadicState = Function.ArgumentsHelper.getState(frame.getArguments());
+        monadicState = null; // Function.ArgumentsHelper.getState(frame.getArguments());
```

Comment:
`monadicState` field is now unused. Remove it altogether.

Reply:
Removed in a62c3c0fb4